### PR TITLE
Team Page Carousel

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Add carousel and very sick animation
- Also added the routing to the view
image
image

### CODE REVIEW

**Comment by daniel-panhead** [comment]
ugly

File: src/components/ProfileCard.vue
Comment (daniel-panhead) [inline src/components/ProfileCard.vue:1]: I would make the mouse off transition duration the same as the mouse on transition duration - bit too slow to transition out

**Review by daniel-panhead — APPROVED** [review]
very sexy

File: src/components/ProfileCard.vue
Comment (IcedTea2K) [reply src/components/ProfileCard.vue:1]: Yes dad

**Pull request merged**